mavros/src/plugins/setpoint_attitude.cpp

```diff
@@ -46,26 +51,46 @@ class SetpointAttitudePlugin : public plugin::PluginBase,
 
 		bool tf_listen;
 
+		typedef message_filters::sync_policies::ApproximateTime<geometry_msgs::PoseStamped, mavros_msgs::Thrust> SyncPoseThrust;
+		typedef message_filters::sync_policies::ApproximateTime<geometry_msgs::TwistStamped, mavros_msgs::Thrust> SyncTwistThrust;
```

Comment:
Use `using` instead `typedef`.
```cpp
using SyncPoseThrust = mesage_filters::sync_polices::ApproximateTime<geometry_msgs::PoseStamped, mavros_msgs::Thrust>;
```

Also may be good to move out of method, e.g. place in `std_plugins::`.